packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -17,7 +17,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    %{mainver}
+Version:    16.3.1
```

Comment:
The last time I did this Ewoudl asked me to keep this, not sure why tho. The script does this automatically...

Reply:
If we keep it as `%{mainver}` it fails at the `Lint Spec` stage in Jenkins.